src/main/resources/twitter4j.properties

```diff
@@ -0,0 +1,6 @@
+
+#oauth.consumerKey = *****
```

Comment:
Why is this commented?

Reply:
Token and keys should not be disclosed so created these credentials with dummy values and commented it.
So while pushing the original credentials will be deleted.

Reply:
Then you can have these uncommented and add a single text comment informing the person who clones this branch to fill in the necessary oauth details in this property file